src/repart/repart.c

```diff
@@ -202,6 +202,7 @@ static AppendMode arg_append_fstab = APPEND_NO;
 static char *arg_generate_fstab = NULL;
 static char *arg_generate_crypttab = NULL;
 static Set *arg_verity_settings = NULL;
+static int arg_relax_copy_block_security = -1;
```

Comment:
I do not think it is necessary to make it tristate, as `-1` and `0` behaves the same. Please use boolean.